lib/superagent-mock.js

```diff
@@ -9,11 +9,16 @@ module.exports = mock;
 
 /**
  * Installs the `mock` extension to superagent.
+ * @param superagent superagent instance
```

Comment:
(minor) Could we have an uppercase letter to mark the separation between the variable name and the variable description?